R/sensorreading-data.R

```diff
@@ -264,17 +235,17 @@ getAppliedCorrection <- function(raw, corrected) {
 #' 
 #' @param value The reading value
 #' 
-#' @param nearestCorrectedValue The nearest corrected value to the reading value
+#' @param nearestcorrectedValue The nearest corrected value to the reading value
 #'
 #' @param uncertainty The uncertainty specified for the reading value
 #' 
 #' @return Yes or No if the corrections applied are within the reference
 #' values for their specified uncertainty
 #' 
-getCorrectedRef <- function (value, nearestCorrectedValue, uncertainty) {
-  if ((!isEmpty(value)) && (!isEmpty(uncertainty)) && (!isEmpty(nearestCorrectedValue))) {
+getCorrectedRef <- function (value, nearestcorrectedValue, uncertainty) {
```

Comment:
I'll make a future task for this, I don't think it's urgent and it would be good to do them together when we don't have a hotfix floating around? @thongsav-usgs